ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-master.yaml

```diff
@@ -156,6 +162,8 @@ tests:
         name: pipeline:logging-elasticsearch6
       - env: IMAGE_LOGGING_KIBANA6
         name: pipeline:logging-kibana6
+      - env: IMAGE_LOG_FILE_METRIC_EXPORTER
```

Comment:
not required here as it is not needed for EO testing

Reply:
fixed, thx

Reply:
fixed